src/HotelCalifornia/Tracing.hs

```diff
@@ -61,8 +63,17 @@ initializeTracing = do
 globalTracer :: MonadIO m => m Tracer
 globalTracer = getGlobalTracerProvider >>= \tp -> pure $ makeTracer tp "hotel-california" tracerOptions
 
+inSpan' :: (MonadUnliftIO m, HasCallStack) => Text -> (Span -> m a) -> m a
+inSpan' spanName action = do
+    tr <- globalTracer
+    Trace.inSpan'' tr spanName defaultSpanArguments action
```

Comment:
the `inSpan''` function does not attach function information, which is fine - we don't really want that showing up in the logs for this